src/main/kotlin/snyk/container/KubernetesWorkloadImage.kt

```diff
@@ -0,0 +1,27 @@
+package snyk.container
+
+import com.intellij.openapi.vfs.VirtualFile
+
+data class KubernetesWorkloadImage(val image: String, val file: VirtualFile) {
```

Comment:
don't data class already has overided fun below?

Reply:
Yeah. Didn't know that. But seems to be the case if you compare all constructor fields.